setup.cfg

```diff
@@ -0,0 +1,52 @@
+[metadata]
+name = broadbean
+version = 0.10.0
```

Comment:
I think we should remove this `version` line as the version info is created by version in git now.

Reply:
For reference when this info is eventually moved to pyproject.toml this should be marked as dynamic. See https://github.com/QCoDeS/Qcodes/pull/4049/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R59 for an example